arch/ARM/ARMMappingInsn.inc

```diff
@@ -154,13 +154,13 @@
 {
 	ARM_BLX, ARM_INS_BLX,
 #ifndef CAPSTONE_DIET
-	{ ARM_REG_PC, 0 }, { ARM_REG_LR, 0 }, { ARM_GRP_CALL, ARM_GRP_V5T, 0 }, 0, 1
+	{ ARM_REG_PC, 0 }, { ARM_REG_LR, 0 }, { ARM_GRP_CALL, ARM_GRP_V5T, 0 }, 1, 1
```

Comment:
BLX is indirect branch, so `0, 1` here is correct.

you can find the definition of these fields at https://github.com/aquynh/capstone/blob/master/utils.h#L22-L23

the rest instructions below have similar issue, so please fix that.